### PR TITLE
Adds NameSlug to decode vin response

### PR DESCRIPTION
**Pull request by KevinJoiner**
Retrieves NameSlug by joining with the device definition table.

### CODE REVIEW

**Review by JamesReate — APPROVED** [review]

File: internal/core/queries/decode_vin.go [hunk 1]
```diff
@@ -106,8 +106,10 @@ func (dc DecodeVINQueryHandler) Handle(ctx context.Context, query mediator.Messa
 		return nil, errors.Wrap(err, "error when beginning transaction")
 	}
 	defer txVinNumbers.Rollback() //nolint
-
-	vinDecodeNumber, err := models.FindVinNumber(ctx, txVinNumbers, vin.String())
+	vinDecodeNumber, err := models.VinNumbers(
+		models.VinNumberWhere.Vin.EQ(vin.String()),
+		qm.Load(models.VinNumberRels.DeviceDefinition)).
+		One(ctx, dc.dbs().Reader)
```
Comment (elffjs) [inline internal/core/queries/decode_vin.go:112]: Should we keep using `txVinNumbers` here?

File: pkg/grpc/decoder.proto [hunk 2]
```diff
@@ -19,6 +19,7 @@ message DecodeVinResponse {
   int32 year = 4;
   string source = 5;
   string powertrain = 6;
+  string name_slug =7;
```
Comment (elffjs) [inline pkg/grpc/decoder.proto:22]:
```suggestion
  string name_slug = 7;
```

File: internal/core/queries/decode_vin.go (hunk 1 above)
Comment (KevinJoiner) [reply internal/core/queries/decode_vin.go:112]: I was wondering if we need a transaction at all here since we are checking for existence and creating if it doesn't exist. Since there is nothing to rollback if a create fails.

Comment (KevinJoiner) [reply internal/core/queries/decode_vin.go]: It doesn't hurt so I will just use it, but I am curious if you can think of an edge case I am missing.

**Comment by elffjs** [comment]
Test failure just looks like Docker stuff. Rerunning.
